features/FEATURE_BLE/targets/TARGET_Maxim/MaximGattServer.cpp

```diff
@@ -146,6 +146,12 @@ ble_error_t MaximGattServer::addService_(GattService &service)
         currAtt->pLen = p_char->getValueAttribute().getLengthPtr();
         currAtt->maxLen = p_char->getValueAttribute().getMaxLength();
         currAtt->settings = ATTS_SET_WRITE_CBACK | ATTS_SET_READ_CBACK;
+
+        if(p_char->getValueAttribute().hasVariableLength())
+        {
```

Comment:
should follow the style in this file `{` on the same line as `if`

Reply:
@0xc0170 Good catch, fixed. :)